dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/engine/command/impl/DeleteDataFileCommand.java

```diff
@@ -85,7 +86,7 @@ protected void executeImpl(CommandContext ctxt) throws CommandException {
 
             if (FileUtil.isPackageFile(doomed)) {
                 try {
-                    String datasetDirectory = doomed.getOwner().getFileSystemDirectory(System.getProperty("dataverse.files.directory")).toString();
+                    String datasetDirectory = doomed.getOwner().getFileSystemDirectory(SystemConfig.FILES_DIRECTORY).toString();
```

Comment:
`SystemConfig.FILES_DIRECTORY` is JUST a name of property. Use method that returns VALUE of property